src/TTGO_T-Beam_LoRa_APRS.ino

```diff
@@ -832,19 +847,35 @@ void loop() {
       delay(300);
       time_delay = millis() + 1500;
       if(digitalRead(BUTTON)==HIGH){
-        if(gps_state == true && gps.location.isValid()){
-            writedisplaytext("((MAN TX))","","","","","");
-            sendpacket();
-        }else{
-            writedisplaytext("((FIX TX))","","","","","");
-            sendpacket();
+        if (!tempOled && enabled_oled) {
+        enableOled(); // turn ON OLED temporary
+        } else {
+          if(gps_state == true && gps.location.isValid()){
+              writedisplaytext("((MAN TX))","","","","","");
+              sendpacket();
+          }else{
+              writedisplaytext("((FIX TX))","","","","","");
+              sendpacket();
+          }
```

Comment:
if OLED is ON the first push of the button will enable the OLED, the second one will trigger the send function